File: 23Even Odds/tempCodeRunnerFile.cpp
#include<iostream>

using namespace std;

int main() {
    long long int n, k;
    cin >> n >> k;
    if(k<=n/2) {
        cout <<2*k-1<<endl;//2n - 1 sobsomoy odd
    }
    else{
        k=k-n/2;
        cout<<2*k<<endl;//2n sobsomoy even
    }
}

File: 23Even Odds/tempCodeRunnerFile.py
n, k = map(int, input("").split( ))
main=[]
odd=[]
even=[]
for i in range(0, n):
    if i%2==0:
        even.append(i)
    else:
        odd.append(i)


main= odd+even

print(main[k-1])